specification/cognitiveservices/data-plane/TranslatorText/stable/v1.1/TranslatorBatch.json

```diff
@@ -68,56 +39,6 @@
                 "format": ""
               }
             }
-          },
```

Comment:
It requires at least one "default" for error response.

Reply:
when adding an error example, the ModelValidation step complains that a long running operation can only contain 202 response code

Reply:
please refer to this validation instance as an example of that
https://github.com/Azure/azure-rest-api-specs/pull/16705/checks?check_run_id=4218187302

Reply:
Added a "default" example. thanks